tools/pagerduty-migrator/migrator/oncall_api_client.py

```diff
@@ -21,6 +21,8 @@ def api_call(method: str, path: str, **kwargs) -> requests.Response:
             cooldown_seconds = int(e.response.headers["Retry-After"])
             sleep(cooldown_seconds)
             return api_call(method, path, **kwargs)
+        elif e.response.status_code == 400:
+            pass
```

Comment:
I think it would be better to handle this at the schedule level instead, being more specific/explicit (here it seems like you will be passing on any 400 API response, which may not be what we want).

Reply:
The way it is today when it already exists, it has broken the code.
My opinion would be.

If it already exists, skip to the next one.